packages/reaction-shipping/server/methods.js

```diff
@@ -44,9 +50,10 @@ Meteor.methods({
   removeShippingMethod: function (providerId, removeDoc) {
     check(providerId, String);
     check(removeDoc, Object);
-    if (!Roles.userIsInRole(Meteor.userId(), ['admin', 'shipping'])) {
-      return false;
+    if (!ReactionCore.hasPermission("shipping")) {
+      throw new Meteor.Error(403, "Access Denied");
     }
```

Comment:
Normally, customers can't get this error, because this errors happens only if someone tries to run methods without admin access. Through console ofc. We could display to him "Are you hacker or something...?" :)